apps/web/src/lang/fr.json

```diff
@@ -8159,5 +8159,21 @@
   "Td1lSw": {
     "defaultMessage": "Il semble que vous n’ayez pas encore ajouté d’expérience de formation à votre curriculum vitæ .",
     "description": "Alert message informing user to add education experience in application education page."
+  },
+  "60tHBx": {
+    "defaultMessage": "Obtention réussie d’un diplôme d’études secondaires ou Programme d’évaluation en education Générale (PEG) equivalent.",
```

Comment:
✍️ @gobyrne I don't think the translation of GED is accurate. GED is a registered trademark and even on the French version of their own site, they don't translate GED: https://ged.com/fr-ca/

For example, the Government of Ontario does not translate the acronym: https://www.ontario.ca/fr/page/apprentissage-des-adultes-certificat-dequivalence-du-cours-secondaire-ged

I can only find one reference to Programme d’évaluation en education Générale (PEG) and it is from an old Government of New Brunswick website:
https://archives.gnb.ca/lhtnb/Resources/Solidarity_Awards_fr-CA.aspx

Here are a few partial strings that include GED that we are currently using:

## Example 1
- equivalent (a GED)
- l’équivalent (formation générale)

## Example 2
- GED equivalent
- l’équivalent (formation générale)

In any case, interested to hear your opinion or if you have another suggestions.

Reply:
I agree with Matt, please follow the usage of the Ontario government.

Reply:
@gobyrne so would the French versions be the following?

## `2osGWe` and `CNDcPx`

- Successful completion of a standard high school diploma or General Education Development (GED) equivalent.
- Obtention réussie d’un diplôme d’études secondaires ou un équivalent du GED.

## `ybi6QM`

- I have a high school diploma or GED equivalent
- Je détiens un diplôme d’études secondaires ou un équivalent du GED

Reply:
It seems like their official site never uses the full spelling and always just puts "GED" in both languages. I guess we should do the same?

So the English above should be "Successful completion of a standard high school diploma or GED equivalent."

Otherwise I think it's the way we want it.

Reply:
@esizer you'll need to update some ids since the `defaultMessage` has changed, but:

## `2osGWe` and `CNDcPx`

- Successful completion of a standard high school diploma or GED equivalent.
- Obtention réussie d’un diplôme d’études secondaires ou un équivalent du GED.

## `ybi6QM`

- I have a high school diploma or GED equivalent
- Je détiens un diplôme d’études secondaires ou un équivalent du GED

Reply:
Thanks @mnigh and @gobyrne . Updated the strings in [update GED strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6668/commits/3fc37ef8d9f499962d8bd5bc996bda8e0a9050e7)